extensions/libraries/redcore/controller/admin/base.php

```diff
@@ -179,6 +179,13 @@ public function publish()
 							break;
 					}
 
+					foreach ($cid as $tableId)
+					{
+						$table = $model->getTable();
+						$table->load($tableId);
+						RFactory::getDispatcher()->trigger('onAfterStoreRedshopb', array(&$table));
```

Comment:
we dont have Redshopb specific triggers in redCORE, you should add this in vanir table object